src/backend/cdb/motion/ic_proxy_backend.c

```diff
@@ -444,7 +445,7 @@ ic_proxy_backend_connect(ICProxyBackendContext *context, ChunkTransportStateEntr
 	/* message key for a HELLO message */
 	ic_proxy_key_init(&backend->key,					/* key itself */
 					  gp_session_id,					/* sessionId */
-					  gp_command_count,					/* commandId */
+					  MyProc->queryCommandId,			/* commandId */
```

Comment:
Could you add tests for ic proxy?

Reply:
Problem with the IC proxy is reproduced only together with the monitoring extension. So the test will be added to the extension code in a separate task.